README.md

```diff
@@ -1 +1,67 @@
-# ProjectTodo
\ No newline at end of file
+    
+![](https://img.shields.io/badge/Microverse-blueviolet)
+Todo List Website
+This website stores a list of things todo's provided by it's users in the local storage. It allows you to remove a todo as well.
+
+
+## Built With
+
+- HTML and CSS and Javascript and webpack
+## Technologies.
+Visual Studio Code.
+nodejs and npm.
+
+
+## Getting Started
+
+
+
+
+To get a local copy up and running follow these simple example steps.
+Locate the project folder in the Repository.
+Click on the code button and dowload a zip file for the project locally.
+Locate the index.html file and double click it to view webpage.
+
+### Prerequisites
+- Basic HTML & CSS knowledge
+- Basic Javasript knowledge
+Node.js, stylelint, browser for displaying it's content.
+install node package manager(npm): npm helps to install stylelint.
+
+
+### Setup
+git clone https://github.com/spydaspider/webpackProjects
+
+### Usage
+Navigate through the web page to find the information you need. Use the links in the pages to move to different sections.
+
+
+### Deployment
+  Project reside in this Repo. You can clone the repo or download the files manually to your local machine.
+
+### Live Demo
+
+## Authors
+
+👤 **Dickson Amankwaah**
+
+- GitHub: [@githubhandle](https://github.com/spydaspider/Micronaut-Dickson-Amankwah)
```

Comment:
Great job @spydaspider 🤝

- [ ] [Optional] You may kindly link this author to the correct GitHub handle.

Reply:
Sure, thank you Mr Gordon for your time.